.github/workflows/AddPRComment.yml

```diff
@@ -5,25 +5,24 @@ on:
       - dev
       - release
 
-permissions:
-  pull-requests: write
+permissions: {}
 
 jobs:
   thank-user:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     name: Say thanks for the PR
     steps:
-      - name: get message
+      - name: Comment on PR
         env:
-          TITLE: ${{ github.event.pull_request.title }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          REPO: ${{ github.repository }}
+          GITHUB_TOKEN: ${{ secrets.CLI_BOT }}
```

Comment:
Recommend use a different token name, or just use ${{ secrets.CLI_BOT }} directly in api call. 
GITHUB_TOKEN is a pre-defined secret from github action